app/routes/application.js

```diff
@@ -1,2 +1,23 @@
 import Route from '@ember/routing/route';
-export default class ApplicationRoute extends Route {}
+import { inject as service } from '@ember/service';
+import { TOAST_OPTIONS } from '../constants/toast-options';
+
+export default class ApplicationRoute extends Route {
+  @service router;
+  @service toast;
+
+  showToast(transition) {
+    this.toast.error(transition.to.queryParams.error, 'Error', TOAST_OPTIONS);
+  }
+
+  beforeModel(transition) {
+    if (
+      transition?.to?.queryParams?.dev === 'true' &&
+      transition?.to?.queryParams?.error
+    ) {
+      if (transition?.to?.queryParams?.error !== '') {
+        this.showToast(transition);
+      }
+    }
+  }
+}
```

Comment:
What is this doing? and also share a small recording of the toast.

Reply:
this is there to prevent user with dev role from using google auth..
should I share clip here in chat or desc?

Reply:
have added it in description as screenshot 2